src/main/java/view/TestModeQuestionView.java

```diff
@@ -216,14 +222,15 @@ public void actionPerformed(ActionEvent evt) {
     private void loadNextQuestion() {
         if (!questions.isEmpty()) {
             currentQuestion = questions.poll();
-            testmodequestion.setText(currentQuestion.getQuestionText());
+            testmodequestion.setText("<html><p style='width:350px;'>" + currentQuestion.getQuestionText()
+                    + "</p></html>");
 
             final List<Answer> answers = currentQuestion.getAnswers();
             final JButton[] options = {option1, option2, option3, option4};
 
             for (int i = 0; i < 4; i++) {
                 if (i < answers.size()) {
-                    options[i].setText(answers.get(i).getAnswerText());
+                    options[i].setText("<html><p style='width:300px;'>" + answers.get(i).getAnswerText());
```

Comment:
Forgot the wrap the text around here. Should've added on</p></html> on the end of the line.